kie-drools-wb/kie-drools-wb-webapp/src/main/resources/org/kie/workbench/drools/KIEDroolsWebapp.gwt.xml

```diff
@@ -70,6 +70,9 @@
   <inherits name="org.drools.workbench.models.commons.DroolsWorkbenchModelsCommon"/>
   <inherits name="org.drools.workbench.models.datamodel.DroolsWorkbenchDataModelAPI"/>
 
+  <!-- Drools Workbench Services -->
+  <inherits name="org.drools.workbench.services.verifier.api.VerifierApi"/>
```

Comment:
Do we need to inherit this here? If you inherit it in [DroolsWorkbenchGuidedDecisionTableEditorClient](https://github.com/droolsjbpm/drools-wb/blob/master/drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/resources/org/drools/workbench/screens/guided/dtable/DroolsWorkbenchGuidedDecisionTableEditorClient.gwt.xml) we should not need it here.